pyproject.toml

```
@@ -27,17 +27,17 @@ classifiers = [
 ]
 keywords = ["sphinx", "extension", "material design", "web components"]
 requires-python = ">=3.7"
-dependencies = ["sphinx>=4,<6"]
+dependencies = ["sphinx>=4,<7"]
 
 [project.urls]
 Homepage = "https://github.com/executablebooks/sphinx-design"
 Documentation = "https://sphinx-design.readthedocs.io"
 
 [project.optional-dependencies]
 code_style = ["pre-commit~=2.12"]
-rtd = ["myst-parser~=0.18.0"]
+rtd = ["myst-parser~=1.0"]
 testing = [
-    "myst-parser~=0.18.0",
+    "myst-parser~=1.0",
```

```suggestion
    "myst-parser>=0.18.0,<2",
```

pinning this to the next major version so that our Sphinx 4 tests don't break